golang/golang.go

```diff
@@ -36,7 +37,7 @@ func SetupGolang(version string) (string, error) {
 	}
 
 	if !isGoInstalled(goDir) {
-		tmp, err := ioutil.TempDir("", fmt.Sprintf("go%s.%s.%s", version, runtime.GOOS, runtime.GOARCH))
+		tmp, err := ioutil.TempDir("", filepath.Base(goDir))
```

Comment:
Like what? Other than the location varying a bit based on the environment (see [`GetTempPath`](https://msdn.microsoft.com/en-us/library/windows/desktop/aa364992(v=vs.85).aspx)) I haven't noticed any issues.